modules/uploading-an-existing-notebook-file-from-a-git-repository-using-jupyterlab.adoc

```diff
@@ -7,7 +7,7 @@
 You can use the JupyterLab user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
 
 .Prerequisites
-* A launched and running Jupyter notebook server.
+* You have a launched and running workbench based on a JupyterLab image.
```

Comment:
@coderabbitai what's actually the difference between launched and running? can you explain?

Reply:
ok then /resolved